classes/Webdashboard/Json.php

```diff
@@ -1,16 +1,51 @@
 <?php
 namespace Webdashboard;
 
+/**
+ * Json class
+ *
+ * JSON functions to read a JSON file or output data in JSON/JSONP format
+ *
+ *
+ * @package Webdashboard
+ */
 class Json
 {
-    /*
-     * Return a json/jsonp representation of data and exits;
+    /**
+     * @var  string  $jsonURI  URI of the JSON stream to read
+     */
+    private $jsonURI;
+
+    /**
+     * Build Json object and set URI (default empty)
+     *
+     * @param  string  $uri  URI of the JSON stream to read
+     */
+    public function __construct($uri = '')
+    {
+        $this->jsonURI = $uri;
```

Comment:
since this is no longer a static function, you can return the object in your setter:
```php
return $this;
```
This way you can chain the outputContent method to the setter (I'll put an example in the model where this is used)